README.md

```diff
@@ -125,19 +125,3 @@ sbt stage
 ```
 The last command prints all available queries - add your own in querydb, run the above commands again to see that your query got deployed.
 More details in the [separate querydb readme](querydb/README.md)
```

Comment:
## Benchmarks

Benchmarks have been moved to https://github.com/joernio/joern-benchmarks

Reply:
I can't access that link btw - is it private?

Reply:
Yes it will be public shortly

Reply:
Seems I don't have permissions to change visibility, have pinged fabs